src/pybind/mgr/dashboard/frontend/e2e/page-helper.po.ts

```diff
@@ -21,6 +21,80 @@ export abstract class PageHelper {
     return $$('.nav.nav-tabs li').count();
   }
 
+  /**
+   * Returns the cell with the exact content given in `content`.
+   *
+   * Limitations: Does not work correctly if the table contains more than 10
+   * entries.
+   */
+  static getTableCellByContent(content: string): ElementFinder {
+    const elem = $('datatable-footer');
+    const count = Number(elem.getAttribute('ng-reflect-row-count'));
+    if (count > 10) {
```

Comment:
What is the behavior of the function if the table contains more than 10 entries, excluding the handler you've included here?

Reply:
The default amount of entries displayed by all tables is 10. The method does not browse through the pages of the table shown and does not utilize the search to find the cell in all entries of the table. That means there's currently a chance to get a false negative in case there are more than 10 entries in the table. The cell with the correct content could be there but if it isn't among the 10 displayed entries, it won't be found. I like to be on the safe side and prefer to just throw an error if there are more than 10 entries in the table instead of having some returned and some failing, although they are there. I'm not fond of the current solution and I will likely fix or at least improve it within this PR.